src/modules/roc_sndio/ibackend.cpp

```diff
@@ -14,5 +14,20 @@ namespace sndio {
 IBackend::~IBackend() {
 }
 
+bool IBackend::add_driver_uniq(core::Array<DriverInfo>& arr, const char* driver_name) {
```

Comment:
IBackend is an "interface class", i.e. all its methods should be pure. Actually, this function does not need to be a member of any class, so I suggest to make it a free function and place it in driver_info.h/driver_info.cpp.